docs/notes/fetch_env_sac_actrepeat_notes.rst

```diff
@@ -0,0 +1,53 @@
+Fetch Environments with SAC + Action Repeat
```

Comment:
Should add this file as one entry in "docs/notes.rst". And probably also add an entry to README.md under "SAC" section.